cypress/integration/example-auto-scroll-when-dragging.spec.js

```diff
@@ -6,6 +6,7 @@ describe('Example - Frozen Columns & Rows', { retries: 1 }, () => {
   const cellHeight = 25;
   const gridWidth = 600;
   const gridHeight = 350;
+  const scrollbarDimension = 17;
```

Comment:
instead of a fixed size, you can get scrollbar dimension directly from SlickGrid
https://github.com/6pac/SlickGrid/blob/master/slick.grid.js#L6059

Reply:
I see, but this code is in the cypree test spec file, I didn't find the way to get the SlickGrid...